src/meta/test/BalancerTest.cpp

```diff
@@ -1174,16 +1145,9 @@ TEST(BalanceTest, CleanLastInvalidBalancePlanTest) {
     CHECK(ok(ret));
     auto balanceId = value(ret);
 
-    sleep(1);
-    // stop the running plan
-    TestUtils::registerHB(kv.get(), {{"0", 0}, {"1", 1}, {"2", 2}});
-    auto stopRet = balancer.stop();
```

Comment:
why don't check the stop result ?

Reply:
Previously the task is delayed by 3 seconds, so need to stop the others task. This ut has been changed, so no need to stop anymore.